packages/accounts-base/accounts_common.js

```diff
@@ -1,55 +1,223 @@
-// @summary Super-constructor for AccountsClient an AccountsServer.
-// @locus Anywhere
-// @param options {Object} an object with fields:
-// - connection {Object} Optional DDP connection to reuse.
-// - ddpUrl {String} Optional URL for creating a new DDP connection.
-AccountsCommon = function _AccountsCommon(options) {
-  // Currently this is read directly by packages like accounts-password
-  // and accounts-ui-unstyled.
-  this._options = {};
-
-  // Note that setting this.connection = null causes this.users to be a
-  // LocalCollection, which is not what we want.
-  this.connection = undefined;
-  this._initConnection(options || {});
-
-  // There is an allow call in accounts_server.js that restricts writes to
-  // this collection.
-  this.users = new Mongo.Collection("users", {
-    _preventAutopublish: true,
-    connection: this.connection
-  });
-
-  // Callback exceptions are printed with Meteor._debug and ignored.
-  this._onLoginHook = new Hook({
-    bindEnvironment: false,
-    debugPrintExceptions: "onLogin callback"
-  });
-
-  this._onLoginFailureHook = new Hook({
-    bindEnvironment: false,
-    debugPrintExceptions: "onLoginFailure callback"
-  });
-};
-
-var Ap = AccountsCommon.prototype;
-
 /**
- * @summary Get the current user id, or `null` if no user is logged in. A reactive data source.
- * @locus Anywhere but publish functions
+ * @summary Super-constructor for AccountsClient and AccountsServer.
+ * @locus Anywhere
+ * @class AccountsCommon
+ * @instancename accountsClientOrServer
```

Comment:
Maybe just "accounts" here? It looks a little silly with such a long name. But it's not a huge deal.